examples/cf-worker-uploader-remover/README.md

```diff
@@ -0,0 +1,30 @@
+# Example: Cloudflare Worker Component
+
+This component allows you to deploy your Serverless Javascript code to a Cloudflare Edge Worker
+
+## Components
+
+TODO -- Below this is unchanged from the basic Example readme. @pinkerton, your documentation skills are needed. Change what you need to
```

Comment:
@pinkerton this is plagiarized, need a README here :) thanks

Reply:
smh